ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagForm.tsx

```diff
@@ -49,7 +49,7 @@ export const TagForm = ({
     formErrors = {
       ...errors,
       tags: errors.name,
-    };
+    } as Record<string, string | string[]>;
```

Comment:
Not sure why this was giving a type error when the file was moved...

Reply:
Strangely if I remove that it doesn't give me an error...